aptos-move/framework/aptos-framework/sources/AptosGovernance.move

```diff
@@ -139,6 +140,9 @@ module AptosFramework::AptosGovernance {
         proposer: &signer,
         stake_pool: address,
         execution_hash: vector<u8>,
+        code_location: String,
+        title: String,
```

Comment:
adding here a `title` means a proposal title would be stored on chain and we can fetch it and display it on the UI?

Reply:
Yes